spring-kafka/src/main/java/org/springframework/kafka/listener/ExceptionClassifier.java

```diff
@@ -185,7 +186,8 @@ public boolean removeNotRetryableException(Class<? extends Exception> exceptionT
 	 * @see #addNotRetryableExceptions(Class...)
 	 * @see #setClassifications(Map, boolean)
 	 */
-	public boolean removeClassification(Class<? extends Exception> exceptionType) {
+	@Nullable
+	public Boolean removeClassification(Class<? extends Exception> exceptionType) {
```

Comment:
Why do we need this change?

Reply:
Oops; thanks!

Reply:
We should change the javadoc to `@return the classification of the exception if the removal was successful; null otherwise.`

I can do it during the merge if you like; but I have to run out for a short while.

Reply:
I can do that, no worries, thanks!

Reply:
Changed the javadoc and added two assertions to the LCFC new methods